CMakeLists.txt

```diff
@@ -58,6 +58,7 @@ option( ENABLE_OPENCL "Build targets dependent on OpenCL?" ON )
 option( ENABLE_ALL "Enable all dependencies and features?" OFF )
 
 if( CMAKE_SYSTEM_NAME MATCHES Linux )
+  option( ENABLE_X11 "Build X11 versions of the targets?" ON)
```

Comment:
I think that ENABLE_X11_DRI3 should depend from ENABLE_X11, use cmake_dependent_option.